scripts/installer/install.py

```diff
@@ -452,15 +446,6 @@
 print("Setting App. Server heap size (Xmx) to "+str(gfHeap)+" Megabytes")
 config.set('glassfish','GLASSFISH_HEAP', str(gfHeap))
 
-# 4b1. PostgresQL driver:
-pg_driver_jarpath = "pgdriver/"+pgJdbcDriver
-
-try:
-   copy2(pg_driver_jarpath, gfJarPath)
-   print("Copied "+pgJdbcDriver+" into "+gfJarPath)
-except:
-   print("Couldn't copy "+pgJdbcDriver+" into "+gfJarPath+". Check its permissions?")
-
```

Comment:
Mostly a question for @landreev , I suppose, but should we continue to maintain the old Perl installer and make equivalent changes there? @poikilotherm I'm talking about `scripts/installer/install`.

Reply:
Yes, I'd like to keep scripts/installer/install working, at least for a short while.

Reply:
Done in 7ed1de7